specs/Qowaiv.Specs/Date_specs.cs

```diff
@@ -155,45 +157,65 @@ public void from_string()
     }
 
     [Test]
-    public void to_string()
-    {
-        using (TestCultures.en_GB.Scoped())
-        {
-            Converting.ToString().From(Svo.Date).Should().Be("11/06/2017");
-        }
-    }
+    public void from_date_time()
+        => Converting.From(new DateTime(2017, 06, 11, 00, 00, 000, DateTimeKind.Local)).To<Date>().Should().Be(Svo.Date);
 
+#if NET6_0_OR_GREATER
     [Test]
-    public void from_DateTime()
-        => Converting.From(new DateTime(2017, 06, 11, 00, 00, 000, DateTimeKind.Local)).To<Date>().Should().Be(Svo.Date);
+    public void from_date_only()
+        => Converting.From(Svo.DateOnly).To<Date>().Should().Be(Svo.Date);
+#endif
 
     [Test]
-    public void from_DateTimeOffset()
+    public void from_date_time_offset()
```

Comment:
I've used both, but think I prefer the latter. Why do you prefer the type name?

Reply:
Because the test is about the class DateTimeOffset. Or do you hint at a separate, more high-level, concept called "date_time_offset", which happens to be implemented by DateTimeOffset?
There might be some ambiguity as well. Consider:
`calculate_DateTime_offset` vs `calculate_date_time_offset` vs `calculate_DateTimeOffset`.

Reply:
I get your point. The problem is, however, that it challenging (if even possible) to do it right all the time. Like: `Date_span_specs.Can_add.two_date_spans()`. Besides the choice made (trough all Qowaiv specs) for the namespace, `Can_add.two_DateSpan_s`, or `Can_add.two_DateSpans`?